src/recommendations/data/hooks/useRecommendations.jsx

```diff
@@ -0,0 +1,20 @@
+import { useSelector } from 'react-redux';
+
+import useAlgoliaRecommendations from './useAlgoliaRecommendations';
+
+/**
+ * This hooks returns Algolia and Static recommendations (Trending and Popular products).
+ * - Static recommendations are always returned.
+ * - Algolia recommendations are returned only if functional cookies are enabled and algolia returns recommendations.
+ * @param educationLevel
+ * @returns {{isLoading: boolean, trendingProducts: *[], popularProducts: *[], algoliaRecommendations: *[]}}
+ */
+export default function useRecommendations(educationLevel) {
```

Comment:
we dont need `useRecommendations` wrapper now, we can directly have country in the `useAlgoliaRecommendations` hook and we can use this hook directly in recommendations component.